manifests/u/UbiquitiInc/UnifiNetworkApplication/8.0.28/UbiquitiInc.UnifiNetworkApplication.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: UbiquitiInc.UnifiNetworkApplication
+PackageVersion: 8.0.28
+InstallerLocale: en-US
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+  Custom: ''
+UpgradeBehavior: install
+Installers:
+- Architecture: x86

```

> The specified executable is not a valid application for this OS platform. 

From the errors, I'm guessing this might be a 64-bit application. Our `x86` validations are performed on 32-bit OSes.

```suggestion
- Architecture: x64
```

---

Bingo, I missed this when checking the YAML. Thank you!